app/src/main/res/values/colors.xml

```diff
@@ -44,6 +44,8 @@
   <color name="color_primary_dark_wtm">#3D9B86</color>
   <color name="color_primary_ioextended">#50B8D0</color>
   <color name="color_primary_dark_ioextended">#30A1BC</color>
+  <color name="color_primary_gcpnext">#E74235</color>
```

Comment:
So here is the API to get the colors of tags https://hub.gdgx.io/api/v1/tags/gcpnext

I am not sure if that is the right color tho. gcpnext and gcp don't have the same colors. How did you choose them @Splaktar ?

And for the darker color for the status bar, we don't have anything, you can use anything you want.